src/simulation/dynamics/_GeneralModuleFiles/THROperation.h

```diff
@@ -33,8 +33,10 @@ THROperation
 //@endcond
 {
     uint64_t fireCounter;                //!< (-) Number of times thruster fired
-    double ThrustFactor;                 //!< -- Current Thrust Percentage
+    double ThrustFactor;                 //!< -- Current thrust scaling factor due to ramp up/down
```

Comment:
Again, no need to have `--` if there are no unit.  Can cause odd HTML display errors in the documentation.

Reply:
Agreed, updating the full file to be consistent with THRSimConfig.h